### PR TITLE
readme: fill in template fields

### PR DESCRIPTION
**Pull request by yar-fed**
Signed-off-by: Yaroslav Fedoriachenko <yar.fed99@gmail.com>

### CODE REVIEW

**Review by sanya190900 — COMMENTED** [review]
change emails

File: README.md [hunk 1]
```diff
@@ -1,28 +1,19 @@
...
-fulfill/rewrite this document (at least, replace everything in `{...}`).
+This is a university project for "Software testing" course labs made by team **Pepega Squad**.
 
 ## Students group
 
-- {Surname Name (in Ukrainian)} {email}
-- ...
-- {Surname Name (in Ukrainian)} {email}
+- Федоряченко Ярослав `yar.fed99@gmail.com`
+- Феофанов Іван `yar.fed99@gmail.com`
```
Comment (Gemofort) [inline README.md:8]: Change email to vanya6677@gmail.com

**Review by Gemofort — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Gemofort) [reply README.md:8]: The same with Alex

**Review by sanya190900 — COMMENTED** [review]
sanya190900@gmail.com

**Review by Gemofort — COMMENTED** [review]
Noice

**Review by Gemofort — APPROVED** [review]

**Pull request merged**